privacyidea/cli/pitokenjanitor/utils/importtokens.py

```diff
@@ -95,3 +97,43 @@ def pskc(pskc_file, preshared_key, validate_mac):
               f" because they could not be validated: {not_parsed_tokens}")
     print(f"Successfully imported {success} tokens.")
     print(f"Failed to import {failed} tokens: {failed_tokens}")
+
+@importtokens_cli.command('privacyidea')
+@click.argument('file', nargs=1, type=click.File())
+@click.argument('key', nargs=1, type=str)
```

Comment:
Similarly to the pskcs import command the key should be given as a parameter  (can be required if we don't want to support plain-text import/export).